assets/javascripts/discourse/components/ai-persona-list-editor.gjs

```diff
@@ -0,0 +1,70 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { LinkTo } from "@ember/routing";
+import { htmlSafe } from "@ember/template";
+import { cook } from "discourse/lib/text";
+import { renderIcon } from "discourse-common/lib/icon-library";
+import I18n from "discourse-i18n";
+import AiPersonaEditor from "./ai-persona-editor";
+
+export default class AiPersonaListEditor extends Component {
+  @tracked _noPersonaText = null;
+
+  constructor() {
+    super(...arguments);
+  }
+
+  get noPersonaText() {
+    if (this._noPersonaText === null) {
```

Comment:
I guess cause translating markdown is easier than translating html. that said, not a hill I would die on.

Reply:
I just think it's a strange pattern to introduce in one place, AFAICT there are no other cases where we use markdown in the translation files. I think in this case the end result is not too complex for translators, and it maintains consistency:

```
<h2>What are AI Personas?</h2>

<p>AI Personas are a powerful feature that allows you to customize the behavior of the AI engine in your Discourse forum. They act as a 'system message' that guides the AI's responses and interactions, helping to create a more personalized and engaging user experience.</p>

<h2>Why use AI Personas?</h2>

<p>With AI Personas, you can tailor the AI's behavior to better fit the context and tone of your forum. Whether you want the AI to be more formal for a professional setting, more casual for a community forum, or even embody a specific character for a role-playing game, AI Personas give you the flexibility to do so.</p>

<h2>Group-Specific Access to AI Personas</h2>

<p>Moreover, you can set it up so that certain user groups have access to specific personas. This means you can have different AI behaviors for different sections of your forum, further enhancing the diversity and richness of your community's interactions.</p>
```